CMakeLists.txt

```diff
@@ -28,6 +28,46 @@ include (ValaVersion)
 ensure_vala_version ("0.34.0" MINIMUM)
 include (ValaPrecompile)
 
+find_package (PkgConfig)
+
+OPTION (WITH_UNITY "Add Unity launcher support" ON)
+pkg_check_modules (UNITY unity>=4.0.0)
+
+if (WITH_UNITY AND UNITY_FOUND)
+    pkg_check_modules (DEPS REQUIRED
```

Comment:
You can create a list of dependencies, and append it "plank" before doing pkg_check_modules(DEPS REQUIRED ${my_list})
So you don't have to rewrite the entire list